svthalia/concrexit
# Add events API v2

Part of #1428

### Summary
Add events API v2

### How to test
Steps to test the changes you made:
1. Check the API docs
2. Check the endpoints for the events
